code/drasil-example/Drasil/Projectile/Concepts.hs

```diff
@@ -31,14 +32,17 @@ projMotion = compoundNC projectile motion
 ---
 
 defs :: [ConceptChunk]
-defs = [launcher, projectile, target]
+defs = [launcher, projectile, target, velVecSpeed, speed1DAcc]
 
-launcher, projectile, target :: ConceptChunk
+launcher, projectile, target, velVecSpeed, speed1DAcc :: ConceptChunk
 launcher   = dcc "launcher"   (nounPhraseSP "launcher")  ("where the projectile is launched from " ++
                                                           "and the device that does the launching")
 projectile = dcc "projectile" (nounPhraseSP "projectile") "the object to be launched at the target"
 target     = dcc "target"     (nounPhraseSP "target")     "where the projectile should be launched to"
 
+velVecSpeed = dccWDS "velVecSpeed" (cn' "vector speed") (S "magnitude" `S.the_ofThe` phrase velocity +:+ phrase vector)
+speed1DAcc  = dccWDS "speed1DAcc"  (cn' "final speed") (S "1D speed under" +:+ phrase constant +:+ phrase acceleration)
```

Comment:
This here is confusing. Why is it called `speed1DAcc`, termed "1D speed under constant acceleration" but labelled "final speed"?